### DISCUSSION TITLE
[Nitpick/Visual]: Hovering over album art in Now Playing bar does not show a flyout with a larger version of the album art as in Cider 1

### DISCUSSION THREAD

**Discussion by Enstrayed** [ciderapp/Cider-2#214] [Feature Requests]
_2023-05-20: The issue formatting has been removed to suit this new 'discussion' format._

In Cider 1, hovering over the album art thumbnail in the now playing bar would reveal a larger version of the album art, which is quite nice when you want to see what the album art looks like without entering immersive mode. This functionality was not replicated in Cider 2.

**Comment by jonybekov** [comment]
Yeah, that would be great!
